fundamentals/middleware/rate-limit/WebRateLimitAuth/SampleRateLimiterPolicy.cs

```diff
@@ -0,0 +1,38 @@
+using System.Threading.RateLimiting;
+using Microsoft.AspNetCore.RateLimiting;
+using Microsoft.Extensions.Options;
+using WebRateLimitAuth.Models;
+
+namespace WebRateLimitAuth;
+
+public class SampleRateLimiterPolicy : IRateLimiterPolicy<string>
+{
+    private Func<OnRejectedContext, CancellationToken, ValueTask>? _onRejected;
+    private readonly MyRateLimitOptions _options;
+
+    public SampleRateLimiterPolicy(ILogger<SampleRateLimiterPolicy> logger,
+                                   IOptions<MyRateLimitOptions> options)
```

Comment:
@Kahbazi I'm now passing in `IOptions` so I can use config for setting the values.